platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
@BogdanAlexandru it confused us too, and we are working to remove that dependency, which we believe is superfluous.